frontend/src/global_styles/layout/_base_mobile.sass

```
@@ -83,3 +83,7 @@
 @media screen and (max-width: $breakpoint-md)
   .hidden-for-tablet
     display: none !important
+
+@media screen and (max-width: $breakpoint-lg) and (min-width: $breakpoint-sm)
+  .hidden-for-mini-laptop
+    display: none !important
```

I would propbably split this up, because it now covers tablet as well as small screens. 
With regards to the naming, I'd emphasize that is a restricting breakpoint by adding `-only`.

```suggestion
@media screen and (max-width: $breakpoint-lg) and (min-width: $breakpoint-md)
  .hidden-for-small-screen-laptops-only
    display: none!

@media screen and (max-width: $breakpoint-md) and (min-width: $breakpoint-sm)
  .hidden-for-tablet-only
    display: none !important
```